assets/src/scss/editorOverrides.scss

```diff
@@ -112,7 +124,7 @@
 }
 
 .editor-rich-text__editable.wp-block-button__link:not([role="textbox"]) {
```

Comment:
This selector doesn't need variables? 🤔

Reply:
I think I tried adding them but it had some difficulties and required further changes, so I kept them out of this PR.

Reply:
I mostly changed this file to fix the incorrect naming for some of the variables, and add some obviously missing ones without going too much into CSS rabbit holes. Created a separate ticket for those https://jira.greenpeace.org/browse/PLANET-6168